services/outputhost/messagecache.go

```diff
@@ -781,6 +786,8 @@ func (msgCache *cgMsgCache) manageMessageDeliveryCache() {
 					msgCache.numAcks = 0
 				}
 			}
+		case <-msgCache.cfgRefreshTicker.C:
```

Comment:
Do you need another ticker ? Can you do this refresh whenever redeliveryTicker fires ? Since the configMgr already caches the object, calling Get() should be light-weight.